From 80f22266bcd157eeb1ce2b4a5725eeccfa1b4aa0 Mon Sep 17 00:00:00 2001
From: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
Date: Tue, 30 May 2023 17:59:47 +0300
Subject: [PATCH] of: enable using OF_DYNAMIC without OF_UNITTEST

Some i.MX drivers use OF_DYNAMIC. Allow the option to be enabled
independently of OF_UNITTEST.

Upstream-Status: Pending
Signed-off-by: Oleksandr Suvorov <oleksandr.suvorov@foundries.io>
---
 drivers/of/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/of/Kconfig b/drivers/of/Kconfig
index da9826accb1b..d9ce37d6ddc6 100644
--- a/drivers/of/Kconfig
+++ b/drivers/of/Kconfig
@@ -67,7 +67,7 @@ config OF_KOBJ
 # Hardly any platforms need this.  It is safe to select, but only do so if you
 # need it.
 config OF_DYNAMIC
-	bool "Support for dynamic device trees" if OF_UNITTEST
+	bool "Support for dynamic device trees"
 	select OF_KOBJ
 	help
 	  On some platforms, the device tree can be manipulated at runtime.
-- 
2.43.0
